client/src/components/Review/DetailsPanel.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import axios from 'axios';
+
+class DetailsPanel extends React.Component {
+    state = {
+      location_id: []
+    }
+  
+    componentDidMount() {
+      axios.get(`https://wheretocode-master.herokuapp.com/reviews/1`)
+        .then(res => {
+          const location_id = res.data;
+          this.setState({ location_id });
+        })
+    }
+  
+    render() {
+      return (
+        <ul>
+          { this.state.location_id.map(location=>
+          <li>UserId: {location.user_id},  Rating: {location.rating}, Comments:{location.comments}</li>)}
```

Comment:
This needs to be rewritten properly. A li tag is not the same as a p tag for accessability purposes. This should look like 
`{ this.state.location_id.map(location => {
    return (
        <ul key={location.id}>
           <li>
               <p>UserId: {location.user_id},</p>
           </li>
           <li>
               <p>Rating: {location.rating},</p>
           </li>
            <li>
                <p>Comments: {location.comment}</p>
            </li>
        </ul>
          )
     }`

Reply:
Updated, thanks.